opennms-container/minion/minion-config-schema.yml

```diff
@@ -123,74 +123,74 @@ categories:
   # Karaf
   - name: "karaf"
     label: "Karaf"
-    description: "TODO"
+    description: "Karaf configuration"
     categories:
       # Karaf :: Management
       - name: "management"
         label: "Management"
-        description: "TODO"
+        description: "Configuration for Karaf JMX management access"
         categories:
           # Karaf :: Management :: RMI
           - name: "rmi"
             label: "RMI"
-            description: "TODO"
+            description: "Configure remote method invocation (RMI)"
             config:
               - name: "registry-host"
                 type:
                   name: "host"
                 label: "Registry Host"
-                description: "TODO"
+                description: "RMI registry host IP address"
                 key: "/karaf/management/rmi/registry/host"
-                example: "TODO"
+                example: "127.0.0.1"
               - name: "registry-port"
                 type:
                   name: "port"
                 label: "Registry Port"
-                description: "TODO"
+                description: "RMI listening TCP port"
```

Comment:
Some descriptions for ports have "listening" mentioned. Should this be removed throughout for consistency?  Conversely, could add "listening" to the others?